### PR TITLE
Update dependencies - Closes #594

### LINKED ISSUE #594: Verify/Update dependencies
In order to stay free of vulnerabilities and improve performance for our application, we need to update dependencies at regular intervals. The 1.0.0 release is the perfect time to do this uplift.

### PR DESCRIPTION
**Pull request by diego-G**
- Package.json updated to the last available modules
- Issue with Chai resolved: new assertion sintax integrated

### CODE REVIEW

**Comment by diego-G** [comment]
I've found another convenient update related to lisk-js module (from 0.2.3 to 0.4.2). Need some more time to check it out before going through. I will need you guys to review again.

File: modules/peers.js [hunk 1]
```diff
@@ -81,12 +81,12 @@ __private.getByFilter = function (filter, cb) {
 			var sort_res =
 				// Nulls last
 				a[field] === b[field] ? 0 :
-				a[field] === null ? 1 :
-				b[field] === null ? -1 :
+					a[field] === null ? 1 :
```
Comment (MaciejBaj) [inline modules/peers.js]: are all of these indentation changes by purpose?

Comment (diego-G) [inline modules/peers.js]: All related to Eslint scan - last commit necessary to pass the test set.

File: sql/blocks.js [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
+		if (params.where.length) {
+			return 'SELECT COUNT("b_id")::int FROM blocks_list WHERE ' + params.where.join(' AND ');
+		} else {
+			return 'SELECT COALESCE((SELECT height FROM blocks ORDER BY height DESC LIMIT 1), 0)';
+		}
+	},
+
+	aggregateBlocksReward: function (params) {
+		return [
+			'WITH',
```
Comment (4miners) [inline sql/blocks.js]: Queries are unreadable without proper intend, can we add exception for files in `sql` directory?

Comment (MaciejBaj) [inline sql/blocks.js]: +1

File: logic/account.js [hunk 3]
```diff
@@ -447,12 +447,12 @@ Account.prototype.removeTables = function (cb) {
...
-				type: 'remove',
-				table: table
-			});
-			sqles.push(sql.query);
+		sql = jsonSql.build({
+			type: 'remove',
+			table: table
 		});
+		sqles.push(sql.query);
+	});
```
Comment (MaciejBaj) [inline logic/account.js]: i would say less readable

**Comment by diego-G** [comment]
Eslint scan is well placed here because it has been raised after updating Mocha and Grunt to the lastest stable version.

**Comment by karmacoma** [comment]
@diego-G: Please update lisk-js to `0.4.3` which was released last week.

**Comment by diego-G** [comment]
@karmacoma is already done on commit #a6a7cda

**Pull request merged**